scripts/install/linux/install-linux.sh

```
@@ -0,0 +1,308 @@
+# Ensure we're in the repository root
+cd "$REPO_ROOT"
+
+if [ ! -f "package.json" ]; then
+    error "package.json not found. Please run this script from the talawa-api repository root."
+    exit 1
+fi
+
+# Detect Linux distribution
+detect_distro() {
+    if [ -f /etc/os-release ]; then
+        . /etc/os-release
```

Shellcheck (0.11.0)

[info] Not following: /etc/os-release was not specified as input (see shellcheck -x).

(SC1091)